service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/RespondentAboutToStartControllerTest.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableMap;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.OverrideAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import uk.gov.hmcts.reform.authorisation.ServiceAuthorisationApi;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.service.RespondentMigrationService;
+import uk.gov.hmcts.reform.idam.client.IdamApi;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@ActiveProfiles("integration-test")
+@WebMvcTest(RespondentController.class)
+@OverrideAutoConfiguration(enabled = true)
+class RespondentAboutToStartControllerTest {
+
+    private static final String AUTH_TOKEN = "Bearer token";
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
```

Comment:
All below are not used and should be removed:

```

    @Autowired
    private RespondentMigrationService respondentMigrationService;

    @MockBean
    private IdamApi idamApi;

    @MockBean
    private ServiceAuthorisationApi serviceAuthorisationApi;
```

Reply:
I was getting test failures without these being set - did have a chat about you with this.

Reply:
I removed these and I am not getting any errors. Can you try again?